tools-local/tasks/GetBuildableFrameworks.cs

```diff
@@ -31,21 +32,20 @@ public override bool Execute()
             List<string> args = new List<string>();
             foreach (var projectJsonPath in ProjectJsonPaths)
             {
-                using (TextReader projectFileReader = File.OpenText(projectJsonPath.ItemSpec))
+                string dir = Path.GetDirectoryName(projectJsonPath.ItemSpec);
```

Comment:
This entire task seems like it is going to cause us trouble. Do we have a better way to get what we need here?